tests/cypress/integration/reportRepair/repairAvailability.spec.js

```diff
@@ -78,4 +83,67 @@ describe('repair availability', () => {
     });
 
   });
+
+  describe('for SOR with only location and problem', () => {
+    const address = '1 Downing Street, London, SW1A 2AA';
```

Comment:
There is a fair amount of duplication between this and the [summary tests](https://github.com/City-of-Lincoln-Council/housing-repairs-online-frontend/blob/main/tests/cypress/integration/reportRepair/summary.spec.js#L12) and I'm going to work on reducing it.

Reply:
This will be addressed independent of this PR.